src/core/runtime.cpp

```diff
@@ -1297,6 +1300,7 @@ SHComposeResult composeWire(const std::vector<Shard *> &wire, SHValidationCallba
   ctx.wire = data.wire;
   ctx.userData = userData;
   ctx.onWorkerThread = data.onWorkerThread;
+  ctx.fullRequired = reinterpret_cast<std::unordered_map<std::string_view, SHExposedTypeInfo> *>(data.requiredVariables);
```

Comment:
Can to cast `reinterpret_cast<decltype(ValidationContext::fullRequired)>(...)` to type copy-past issues